src/main/java/sparksoniq/jsoniq/runtime/iterator/postfix/ObjectLookupIterator.java

```diff
@@ -142,7 +139,7 @@ protected void closeLocal() {
 
     @Override
     public Item nextLocal() {
-        if (_hasNext == true) {
+        if (_hasNext) {
```

Comment:
Can spotless catch these things automatically, or was it manual? Just curious.

Reply:
I don't know if spotless has a configuration for this, but this one was manual

Reply:
Very good. Thanks!